### PR TITLE
Meter availability report docs

### PR DESCRIPTION
**Pull request by MiquelIR**
### Description:

> Add documentation for the TG meter availability report
> Improve the organization of titles on the "telegestio" docs page.

### Tasks:

- [ ] Add docs
- [x] Modify tittle organization

### Links:

> Add links for each file modified

* [telegestio #Informe de disponibilitat de comptadors](http://builds.gisce.net/powerp-docs/powerp_meter_availability_report_docs/ca/distri/telegestion/#informe-de-disponibilitat-de-comptadors)
* [telegestion #Informe de disponibilidad de contadores](http://builds.gisce.net/powerp-docs/powerp_meter_availability_report_docs/es/distri/telegestion/#informe-de-disponibilidad-de-contadores)

### CODE REVIEW

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_meter_availability_report_docs/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_meter_availability_report_docs/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

File: docs/distri/telegestion.md [hunk 1]
```diff
@@ -269,12 +271,12 @@ Les excepcions que es gestionen actualment són:
...
     a què utilitzi un altre període.
 
-### Configuració > Excepcions Comptador
+**Configuració > Excepcions Comptador**
 
-En aquest llistat podem indicar els comptadors que validen sempre les seves
-lectures, encara que tinguin errors. Això ens serveix per comptadors de proves,
-comptadors no associats a cap polissa, comptadors de supervisió, comptadors no
-donats d'alta a l'ERP, etc...
+En el llistat **Configuració>Excepcions Comptador** podem indicar els
```
Comment (jaumef) [inline docs/distri/telegestion.md]: ES: Tras la negrita no hay espacio

File: docs/distri/telegestion.md [hunk 2]
```diff
@@ -768,3 +765,41 @@ que es farà la petició.
...
+3. Una taula de targetes de colors en la qual cada targeta fa referència a una hora concreta
+d'un dia de la setmana.
+    * Files: Cada fila representa un dia de la setmana. De dilluns fins diumenge.
+    * Columnes: Cada columna és una hora del dia. L'hora 0 és les 12 de la nit.
+
+4. Una llegenda indicant l'escala de colors utilitzada en les targetes de la taula. Cada color té
+assignat un rang de valors. Aquests valors són la quantitat de connexions establertes correctament
+per el comptador durant el període de temps que abasti l'informe.
+
+    !!! Tip "Exemple"
```
Comment (jaumef) [inline docs/distri/telegestion.md]: ES, no está como nota

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_meter_availability_report_docs/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_meter_availability_report_docs/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Pull request merged**